test/test_smtlib.cpp

```diff
@@ -38,7 +38,7 @@ TEST(TestSMTLib, TestBasicBooleanAnd) {
 
     const auto smtLibLogic = std::make_unique<smtliblogic::SMTLogicBlock>(false, ss);
 
-    const std::string output = "; Generated by LogicBlocks\n"
+    const std::string output = "; Generated by LogicBlocks 1.3.5\n"

```

Comment:
I don't like this being hard coded in the tests. This means the tests need to be updated for every single release.
Might as well use the Macro here 🤷🏼